src/controller/CHIPDeviceController.h

```diff
@@ -393,7 +405,8 @@ class DLL_EXPORT DeviceController : public AbstractDnssdDiscoveryController
 
     FabricIndex mFabricIndex = kUndefinedFabricIndex;
 
-    bool mRemoveFromFabricTableOnShutdown = true;
+    bool mRemoveFromFabricTableOnShutdown;
```

Comment:
These get initialized in `Init()`, so no need to repeat any defaults here.

Reply:
Seems like good practice though .... please keep initializing them.

Reply:
I don’t particularly like having the defaults repeated in 3 places, but Ok.